### PR TITLE
openqa_bootstrap: Select job details tab

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/66111
Verification: https://openqa.opensuse.org/tests/1257157

### CODE REVIEW

File: tests/openqa/webui/test_results.pm [hunk 1]
```diff
@@ -57,6 +57,9 @@ sub run {
     for (1 .. 25) {
         send_key 'f5';
         wait_still_screen;
+        send_key 'home';
```
Comment (ggardet) [inline tests/openqa/webui/test_results.pm:60]: What does this `home` key?

Comment (ggardet) [reply tests/openqa/webui/test_results.pm:60]: Replying to myself: It is to go up on the page. Maybe worth to add a commentary?

**Comment by Zaoliang** [comment]
LGTM

File: tests/openqa/webui/test_results.pm [hunk 2]
```diff
@@ -57,6 +57,9 @@ sub run {
...
+        assert_and_click 'openqa-job-details';
+        wait_still_screen;
```
Comment (okurz) [inline tests/openqa/webui/test_results.pm:62]: This is a horrible waste of time :( I recommend you never add a `wait_still_screen` without argument. At best you can completely replace it with `assert_screen` or `assert_and_click` calls

**Pull request merged**